### DISCUSSION TITLE
Extending a base config

### DISCUSSION THREAD

**Discussion by savage-alex** [apideck-libraries/portman#230] [Ideas]
Would it be possible to have a feature like spectral that extends a core / base config.
That way you can have a generic setup that is centrally located and maintained for all projects and then the project specific path / overwrites ?

**Comment by thim81** [comment]
@savage-alex To better understand your use-case, what would you like to see centralised? 
The CLI configuration or the Portman test suite settings? And would you have an example in mind how that would look?

**Reply by savage-alex** [reply]
Sorry for the slow reply on this one. Ill get an example put together.
